docs/adr/0006-treat-rollouts-differently.md

```diff
@@ -0,0 +1,44 @@
+# Opting out of studies should not affect rollouts
+
+> Status: Open (Pending input from Trust & Legal regarding the addition of a new preference)
+
+> Author: Travis Long
+
+> Deciders: Nimbus team, Trust & Legal
```

Comment:
Should this also include any of the following?
* Desktop Firefox product team members
* Mobile Firefox product team members
* Release management
* Data team members

Reply:
Yes, those are all probably reasonable stakeholders affected by this. I'll add them to the decider list